src/forms.ts

```diff
@@ -78,6 +79,22 @@ export class Forms {
       data,
     })
   }
+
+  public async copy(args: {
+    uid: string
+    workspaceHref: string
+  }): Promise<Typeform.Form> {
+    const { uid, workspaceHref } = args
+    const input = await this.get({ uid })
+
+    const data = removeFormIds(input) as Typeform.Form
```

Comment:
Block `integrations` should be deleted as well

Reply:
Are you sure it is returned by the API? 🤔 [It is not in the `Typeform.Form` type](https://github.com/Typeform/js-api-client/blob/main/src/typeform-types.ts#L299-L382) (could be missing), [I dont see it in docs](https://www.typeform.com/developers/create/reference/retrieve-form/) (could be missing) and it is not being returned for my form that contains some integrations.